ios/FluentUI.Demo/FluentUI.Demo/Demos/BottomCommandingDemoController.swift

```diff
@@ -0,0 +1,160 @@
+//
+//  Copyright (c) Microsoft Corporation. All rights reserved.
+//  Licensed under the MIT License.
+//
+
+import Foundation
+import FluentUI
+
+class BottomCommandingDemoController: DemoController {
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+
+        let bottomCommandingVC = BottomCommandingController()
+        bottomCommandingVC.heroItems = heroItems
+        bottomCommandingVC.expandedListSections = expandedListSections
+
+        addChild(bottomCommandingVC)
+        view.addSubview(bottomCommandingVC.view)
+        bottomCommandingVC.didMove(toParent: self)
+
+        bottomCommandingController = bottomCommandingVC
+
+        container.addArrangedSubview(createLabelAndSwitchRow(labelText: "Expanded list items", switchAction: #selector(toggleExpandedItems), isOn: true))
```

Comment:
instead of this, which doesn't make the demo app a11y friendly in large text mode, can you use tableview with toggles for better larger text support similar to sidetabbardemocontroller please? we are slowly cleaning up this bad pattern in the demo app.

Reply:
Yeah it looks much better now :D